projects/Mallard/src/download-edition/download-and-unzip.ts

```diff
@@ -155,6 +163,10 @@ const runDownload = async (issue: IssueSummary, imageSize: ImageSize) => {
         errorService.captureException(error)
         updateListeners(localId, { type: 'failure', data: error })
```

Comment:
Can we have a better message here?

Reply:
@jimhunty  'updateListeners` has a fixed data type to report updates. Do you mean to provide more info in `data` field?